docs/integrations.md

```diff
@@ -70,6 +70,21 @@ jobs:
 This will default to using the [`r2c` rule pack](https://semgrep.live/p/r2c).
 To choose another rule pack see https://semgrep.live/packs.
 
+Another way to do it is to use the official Semgrep [Docker Image](https://hub.docker.com/r/returntocorp/semgrep)
```

Comment:
Is there a particular advantage to using one vs. the other? Or why list this one after the circleci/python example?